run.py

```diff
@@ -454,7 +455,8 @@ def fit(self):
 	parser.add_argument('-gcn_drop',	dest='dropout', 	default=0.1,  	type=float,	help='Dropout to use in GCN Layer')
 	parser.add_argument('-hid_drop',  	dest='hid_drop', 	default=0.3,  	type=float,	help='Dropout after GCN')
 	parser.add_argument('-disable_gnn_encoder', dest='disable_gnn_encoder', default=False, type=bool, help='Disables the GNN encoder layer')
-
+	parser.add_argument('-enable_early_stopping', dest='enable_early_stopping', default=False, type=bool, help='Enables early stopping with after the amount of epochs that were set.')
+	parser.add_argument('-set_early_stopping_limit', dest='early_stopping_limit', default=25, type=int, help='Sets the amount of epochs after which early stopping can occurr.')
```

Comment:
I would remove the set of the parameter name and occur has an r to much
```suggestion
	parser.add_argument('-early_stopping_limit', dest='early_stopping_limit', default=25, type=int, help='Sets the amount of epochs after which early stopping can occurr.')
```